Doc/library/tkinter.rst

```diff
@@ -13,12 +13,21 @@
 The :mod:`tkinter` package ("Tk interface") is the standard Python interface to
 the Tk GUI toolkit.  Both Tk and :mod:`tkinter` are available on most Unix
 platforms, as well as on Windows systems.  (Tk itself is not part of Python; it
-is maintained at ActiveState.) You can check that :mod:`tkinter` is properly
-installed on your system by running ``python -m tkinter`` from the command line;
-this should open a window demonstrating a simple Tk interface.
+is maintained at ActiveState.)
+
+Running ``python -m tkinter`` from the command line should open a window
+demonstrating a simple Tk interface, letting you know that :mod:`tkinter` is
+properly installed on your system, and also showing what version of Tcl/Tk is
+installed, so you can read the Tcl/Tk documentation specific to that version.
 
 .. seealso::
 
+   `Tk commands <https://www.tcl.tk/man/tcl8.6/TkCmd/contents.htm>`_
+      Commands without underscores in their name correspond to :mod:`tkinter` or :mod:`tkinter.ttk` widget classes.
+      Change '6' to match the minor version of your Tcl/Tk installation (e.g. '5' for Tcl/Tk 8.5).
```

Comment:
Version 9.0 is near. If write about concrete version, It is worth to write it as '8.6', not '6'.

Reply:
I hope you are right.  "Change "8.6" to match the version of your Tcl/Tk installation."  should work.